matchzoo/models/anmm_model.py

```diff
@@ -0,0 +1,70 @@
+"""An implementation of aNMM Model."""
+
+
+import logging
+
+import keras
+from keras.activations import softmax
+from keras.initializers import RandomUniform
+
+from matchzoo import engine
+
+logger = logging.getLogger(__name__)
+
+
+class ANMMModel(engine.BaseModel):
+    """
+    ANMM Model.
+
+    Examples:
+        >>> model = ANMMModel()
+        >>> model.guess_and_fill_missing_params(verbose=0)
+        >>> model.build()
+
+    """
+
+    @classmethod
+    def get_default_params(cls) -> engine.ParamTable:
+        """:return: model default parameters."""
+        params = super().get_default_params(with_embedding=True)
+        params.add(engine.Param('bin_num', 60))
+        params.add(engine.Param('dropout_rate', 0.1))
+        params.add(engine.Param('left_text_len', 10))
```

Comment:
Maybe the `left_text_len` could be inferred from `self._params['input_shapes']`.

Reply:
Fixed.